README.md

```diff
@@ -1,3 +1,4 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15535780&assignment_repo_type=AssignmentRepo)
 # SE_Day1
 Software Engineering Day1 Assignment
 
```

Comment:
Answer software engineering are: the application of a systematic, disciplined, quantifiable approach to the development, operation, and maintenance of software.

We can also define Software engineering as the process of analyzing user needs and then designing, constructing, and testing end-user applications that will satisfy these needs through the use of software programming languages.